CONTRIBUTING.md

```diff
@@ -192,7 +193,8 @@ feat(api): New endpoint for feature X (#1234)
 
 If you have **breaking changes** in your PR, it is important to note them in the PR
 description but also in the merge commit for that PR.
-When pressing "squash and merge", you have the option to fill out the commit message. Please use that feature to add the breaking changes according to the
+When pressing "squash and merge", you have the option to fill out the commit message.
+Please use that feature to add the breaking changes according to the
 [conventional commit guidelines](https://www.conventionalcommits.org/en/v1.0.0/).
```

Comment:
Would an example be helpful here?

Reply:
as mentioned this is not about the content but fixing the build, and removing the markdown issues within those files